piptools/utils.py

```diff
@@ -380,3 +380,13 @@ def create_install_command():
     from pip._internal.commands import create_command
 
     return create_command("install")
+
+
+def get_trusted_hosts(finder):
+    """
+    Returns a list of trusted hosts from given finder.
```

Comment:
Looking at the Pip src, it returns an iterator:  https://github.com/pypa/pip/blob/0f095af4ba7ac1af2c27024fa539af4c3a45d002/src/pip/_internal/index/package_finder.py#L692-L695

Reply:
Addressed in 7a54db3

Reply:
Almost, I imagine that you'd also want to fix the docstring to be precise.
```suggestion
    Returns an iterable of trusted hosts from a given finder.
```

Reply:
Thanks!